### PR TITLE
[4.0] Update Readme for Api tests

### PR DESCRIPTION
**Pull request by astridx**
Pull Request for Issue https://github.com/joomla/joomla-cms/pull/26722#issuecomment-683141279 .

### Summary of Changes

Update Readme

### Testing Instructions

Review

### CODE REVIEW

File: tests/Codeception/README.md [hunk 1]
```diff
@@ -36,14 +36,21 @@ These are the Joomla 4 API (webservices) tests. To run these tests on OSX you wi
 
 ### Installation
 
-Run a `composer install` in the joomla root directory copy the file `tests/Codeception/api.suite.dist.yml`
-to `tests/Codeception/api.suite.yml`. Then adjust the REST url in the new file and copy the `codeception.yml`
-to the Joomla main directory. 
+1) Run a `composer install` in the joomla root directory 
+2) Copy the file `tests/Codeception/api.suite.dist.yml` to `tests/Codeception/api.suite.yml`. Then adjust the REST url in the new file.
+3) Make sure, that the file `codeception.yml` is in the Joomla main directory.
```
Comment (Quy) [inline tests/Codeception/README.md]: Use `main` or `root` and capitalize Joomla as in 1 for consistency?

Comment (astridx) [reply tests/Codeception/README.md]: Thank you Quy

File: tests/Codeception/README.md [hunk 2]
```diff
@@ -36,14 +36,21 @@ These are the Joomla 4 API (webservices) tests. To run these tests on OSX you wi
 
 ### Installation
 
-Run a `composer install` in the joomla root directory copy the file `tests/Codeception/api.suite.dist.yml`
-to `tests/Codeception/api.suite.yml`. Then adjust the REST url in the new file and copy the `codeception.yml`
-to the Joomla main directory. 
+1) Run a `composer install` in the Joomla main directory
```
Comment (Quy) [inline tests/Codeception/README.md]:
```suggestion
1) Run a `composer install` in the Joomla main directory.
```

File: tests/Codeception/README.md [hunk 3]
```diff
@@ -36,14 +36,21 @@ These are the Joomla 4 API (webservices) tests. To run these tests on OSX you wi
 
 ### Installation
 
-Run a `composer install` in the joomla root directory copy the file `tests/Codeception/api.suite.dist.yml`
-to `tests/Codeception/api.suite.yml`. Then adjust the REST url in the new file and copy the `codeception.yml`
-to the Joomla main directory. 
+1) Run a `composer install` in the Joomla main directory 
+2) Copy the file `tests/Codeception/api.suite.dist.yml` to `tests/Codeception/api.suite.yml`. Then adjust the REST url in the new file.
+3) Make sure, that the file `codeception.yml` is in the Joomla main directory.
```
Comment (Quy) [inline tests/Codeception/README.md]:
```suggestion
3) Be sure that the file `codeception.yml` is in the Joomla main directory. 
```

Comment (astridx) [reply tests/Codeception/README.md]: Thanks

File: tests/Codeception/README.md [hunk 4]
```diff
@@ -36,14 +36,21 @@ These are the Joomla 4 API (webservices) tests. To run these tests on OSX you wi
 
 ### Installation
 
-Run a `composer install` in the joomla root directory copy the file `tests/Codeception/api.suite.dist.yml`
-to `tests/Codeception/api.suite.yml`. Then adjust the REST url in the new file and copy the `codeception.yml`
-to the Joomla main directory. 
+1) Run a `composer install` in the Joomla main directory.
+2) Copy the file `tests/Codeception/api.suite.dist.yml` to `tests/Codeception/api.suite.yml`. Then adjust the REST url in the new file.
```
Comment (brianteeman) [inline tests/Codeception/README.md]:
```suggestion
2) Copy the file `tests/Codeception/api.suite.dist.yml` to `tests/Codeception/api.suite.yml`. Then edit the REST url in the new file to point it to your localhost url.
```

**Comment by brianteeman** [comment]
The thing I am unsure of is
Do you have to have a working joomla install before running the api tests?
Do you have to have followed the earlier instructions to set up the system tests first?

**Comment by astridx** [comment]
Thanks @brianteeman  for watching. A ready set up Joomla environment is necessary (You also change the configuration.php) - but not the system tests. That's how it worked for me.

 I just corrected that.

**Review by wilsonge — APPROVED** [review]

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**